tests/build-conf/beagleboneblack/local.conf

```diff
@@ -234,6 +234,7 @@ BB_DISKMON_DIRS = "\
 # libsdl library available on your build system.
 #PACKAGECONFIG_append_pn-qemu-native = " sdl"
 #PACKAGECONFIG_append_pn-nativesdk-qemu = " sdl"
+PACKAGECONFIG_append_pn-gnutls = " p11-kit"
```

Comment:
when I removed that, and the [above](https://github.com/mendersoftware/meta-mender/pull/1070#discussion_r481980342) the beaglebone build [failed](https://gitlab.com/Northern.tech/Mender/mender-qa/-/jobs/714782658):
```
Parsing recipes...done.
Parsing of 792 .bb files complete (0 cached, 792 parsed). 1339 targets, 57 skipped, 0 masked, 0 errors.
NOTE: Resolving any missing task queue dependencies
ERROR: Nothing PROVIDES 'libp11' (but /builds/Northern.tech/Mender/meta-mender/meta-mender-core/recipes-mender/mender-client/mender-client_git.bb DEPENDS on or otherwise requires it). Close matches:
  libx11
NOTE: Runtime target 'mender-client' is unbuildable, removing...
Missing or unbuildable dependency chain was: ['mender-client', 'libp11']
ERROR: Required build target 'core-image-base' has no buildable providers.
Missing or unbuildable dependency chain was: ['core-image-base', 'mender-client', 'libp11']
Summary: There were 2 ERROR messages shown, returning a non-zero exit code.
```

Reply:
That's because you put the dependency in the core layer. Once you fix that by moving it into the qemu layer, this should work.

Reply:
Yep, what @kacf said :)